packages/utils/src/lib/utils.ts

```diff
@@ -51,7 +54,11 @@ export function distinct<T extends string | number | boolean>(array: T[]): T[] {
   return Array.from(new Set(array));
 }
 
-// === File
+// === Filesystem
+
+export function ensureDirectoryExists(baseDir: string) {
+  if (!existsSync(baseDir)) mkdirSync(baseDir, { recursive: true });
+}
```

Comment:
Should use async APIs, so that it's non-blocking.

Reply:
done